cliquet/initialization.py

```diff
@@ -9,7 +9,7 @@
 try:
     import newrelic.agent
 except ImportError:  # pragma: no cover
-    pass
+    newrelic = None
```

Comment:
Actually not :)

``AttributeError: <module 'newrelic' from '../newrelic/__init__.pyc'> does not have the attribute 'agent'``